src/compiler/parser/html-parser.js

```diff
@@ -59,6 +59,10 @@ const decodingMap = {
 const encodedAttr = /&(?:lt|gt|quot|amp);/g
 const encodedAttrWithNewLines = /&(?:lt|gt|quot|amp|#10);/g
 
+// #5992
+const ignoreFirstLFTagList = makeMap('pre,textarea', true)
+const isIgnoreFirstLf = (tag, html) => tag && ignoreFirstLFTagList(tag.toLowerCase()) && html[0] === '\n'
```

Comment:
Not need to call `toLowerCase()` here, and it seems `<textarea>` will never get into this function since `isPlainTextElement('textarea')` equals `true`.